com.unity.render-pipelines.universal/Runtime/Deprecated.cs

```diff
@@ -46,6 +46,16 @@ public override void FinishRendering(CommandBuffer cmd)
         {
             throw new NotSupportedException(k_ErrorMessage);
         }
+
```

Comment:
Why is it being added in `Deprecated.cs`?

Reply:
Because it is marked as an abstract function and the ForwardRenderer is deprecated but still inherits from ScriptableRenderer so it needs to have the function.